src/Core/Content/Media/Thumbnail/ThumbnailSizeCalculator.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopware\Core\Content\Media\Thumbnail;
+
+use Shopware\Core\Content\Media\Aggregate\MediaThumbnailSize\MediaThumbnailSizeEntity;
+
+/**
+ * @phpstan-type ImageSize array{width: int, height: int}
```

Comment:
the type is exactly the same, IMHO we should always just use a `ImageSize` type

Reply:
The ThumbnailService uses the same array in multiple methods (for instance createNewImage). I could declare the phpstan-type within the ThumbnailService and Import it into the Calculator.